libs/filter_example/src/example_filter.rs

```diff
@@ -101,14 +102,14 @@ impl Filter {
         me.push_str(&data_to_append);
         me.push_str(",");
 
-        if to_return.headers.contains_key(&"properties".to_string()) {
-            to_return
+        if mod_rpc.headers.contains_key(&"properties".to_string()) {
```

Comment:
This looks questionable, can we fix it?

Reply:
to_return is a vector of RPCs, and mod_rpc is the RPC based on the RPC given as input.  Because we are now outputting RPCs to go both to the node and to storage, we need to return a vector instead of an Option<Rpc>.